src/navigator/Navigator.tsx

```diff
@@ -120,8 +122,6 @@ import VerificationLoadingScreen from 'src/verify/VerificationLoadingScreen'
 import WalletConnectSessionsScreen from 'src/walletConnect/screens/Sessions'
 import WalletConnectRequest from 'src/walletConnect/screens/WalletConnectRequest'
 import WebViewScreen from 'src/webview/WebViewScreen'
-import KycLanding from 'src/fiatconnect/KycLanding'
-import KycStatus from 'src/fiatconnect/KycStatus'
```

Comment:
This reminds me, we should let prettier / eslint orgamize imports to avoid this.
Maybe via https://github.com/trivago/prettier-plugin-sort-imports?